src/awkward/operations/ak_to_categorical.py

```diff
@@ -30,17 +30,17 @@ def to_categorical(array, *, highlevel=True, behavior=None):
         >>> array = ak.Array([["one", "two", "three"], [], ["three", "two"]])
         >>> categorical = ak.to_categorical(array)
         >>> categorical
-        <Array [['one', 'two', ... 'three', 'two']] type='3 * var * categorical[type=str...'>
-        >>> ak.type(categorical)
+        <Array [['one', 'two', 'three'], ..., [...]] type='3 * var * categorical[ty...'>
+        >>> categorical.type.show()
         3 * var * categorical[type=string]
-        >>> ak.to_list(categorical) == ak.to_list(array)
+        >>> categorical.tolist() == array.tolist()
```

Comment:
Should we promote `to_list` over `tolist`?
```suggestion
        >>> categorical.to_list() == array.to_list()
```